public/docs/features.md

```diff
@@ -11,121 +10,128 @@ If you experience any _issues_, feel free to report it on [**GitHub**](https://g
 Or meet us on [**Matrix.org**](https://riot.im/app/#/room/#codimd:matrix.org) for dev-talk and interactive help.
 **Thank you very much!**
 
-Workspace
-===
-## Modes
-**Desktop & Tablet**
+## Workspace
+
+### Modes
+
+#### Desktop & Tablet
 
 <i class="fa fa-edit fa-fw"></i> Edit: See only the editor.
 <i class="fa fa-eye fa-fw"></i> View: See only the result.
 <i class="fa fa-columns fa-fw"></i> Both: See both in split view.
```

Comment:
Does anyone else wish they were in the order of the toolbar? I know that it's somewhat confusing to talk about "both" when you haven't introduced both, but somehow this order bugs me.

Suggestion:

```diff
-See both in split view.
+See editor and result at the same time.
```
(and then put it as the middle bullet point?)